packages/iTwinUI-react/src/core/ComboBox/ComboBox.tsx

```diff
@@ -65,9 +65,10 @@ export type ComboBoxProps<T> = {
   dropdownMenuProps?: PopoverProps;
   /**
    * Message shown when no options are available.
+   * If `React.ReactNode` is provided, it will be rendered as is and won't be wrapped with `MenuExtraContent`.
```

Comment:
Comment kinda confusing, cause `ReactNode` is also string, right?

Reply:
Suggestion?

Reply:
I would say if JSX.Element is provided, then.....
or maybe just React element?

Reply:
Changed